model/pomoc.xtpl

```diff
@@ -12,8 +12,8 @@
   <label for="chci" class="shopPomoc_label">Chci se zapojit</label>
 
   <div class="shopPomoc_text">
-    GameCon připravujeme celý rok, a tak vždycky oceníme další pár rukou a to nejen přes rok ale i na místě v průběhu
-    festivalu. Ať už se jedná o uvedení nějaké aktivity, přípravu zázemí nebo např. s chodem na Infopultu. Za každou
+   GameCon připravujeme celý rok, a tak vždycky oceníme další pár rukou, a to nejen přes rok ale i na místě v průběhu
+    festivalu. Ať už se jedná o uvedení nějaké aktivity, přípravu zázemí nebo například chod na Infopultu. Za každou
```

Comment:
```suggestion
    festivalu. Ať už se jedná o uvedení nějaké aktivity, přípravu zázemí nebo například chod Infopultu. Za každou
```
Takhle mi to dává asi ještě trochu víc smysl

Reply:
Text navrhovala Danča. Takže můžeš to s ní probrat na trellu.